tasks/go.py

```diff
@@ -78,11 +78,16 @@ def lint_module(target):
             concurrency_arg = "" if concurrency is None else f"--concurrency {concurrency}"
             tags_arg = " ".join(sorted(set(tags)))
             timeout_arg_value = "25m0s" if not timeout else f"{timeout}m0s"
-            return ctx.run(
+            res = ctx.run(
                 f'golangci-lint run {verbosity} --timeout {timeout_arg_value} {concurrency_arg} --build-tags "{tags_arg}" --path-prefix "{module_path}" {golangci_lint_kwargs} {target}/...',
                 env=env,
                 warn=True,
             )
+            # early stop on SIGINT: exit code is 128 + signal number, SIGINT is 2, so 130
+            # for some reason this becomes -2 here
```

Comment:
😆

Reply:
When i run golangci-lint directly it stops with 130, but here invoke returns -2 !

Reply:
I could understand if it was the same for the go binary but for that one the value is really 130 in invoke...